src/Services/LoggingService.php

```diff
@@ -104,6 +116,8 @@ private function getPlentyMarketLoggerInstance(string $method, string $reference
   }
 
   /**
+   * Maps data from the PlentyMarket designed associative array to values of an array to be used by our different log methods
```

Comment:
For some reason, the previous team used "PlentyMarket" but we should always call it "PlentyMarkets" or "PM" for short.